data.csv

```diff
@@ -1,5 +1,5 @@
 salary,isRemote,companyHQ,companySize,jobType,jobRole,programmingLanguage,yearsOfExperience,seniorityLevel,gender,age,isEUCitizen,raceOrEthnicity,education
```

Comment:
It seems like what I forgot to add was the header: `company`. You are welcome to add that instead.